src/features/todolist/AddTodoForm.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import { TodoList } from "./TodoList";
+import { useDispatch } from "react-redux";
+import { addTodos } from "./TodoSclice";
+
+export function AddTodoForm() {
+  const [value, setValue] = useState("");
+
+  const dispatch = useDispatch();
+
+  const onSubmit = (event) => {
+    event.preventDefault();
+    setValue("");
+    dispatch(
+      addTodos({
+        title: value,
```

Comment:
`setValue` assign an empty string to the value, so there is a chance that `value` here will be equal to `''`.